### DISCUSSION TITLE
Did it stop supporting ?

### DISCUSSION THREAD

**Discussion by Vivasat** [joqueka/bf4py#19] [Q&A]
Hey there pal, I have been trying to run some Monte Carlo Simulations but the problem lies with finding right data, I hoped to use your package to retrieve the data right from the Boerse Frankfurt page using your package into my model. But for some reason, it doesn't work. Would really appreciate if we could figure out why ?!

**Comment by joqueka** [comment]
Hi 
please give me more information which functions don't work properly. Those I'm using are working fine.
